GeorgianMemenator/Scenes/DogBreedDetail/DogBreedDetailView.swift

```diff
@@ -14,11 +14,23 @@ struct DogBreedDetailView: View {
         VStack {
             if viewModel.hasError {
                 Text("Error")
-                    .font(.title)
-                    .padding(.top, 32)
             } else if viewModel.isLoading {
                 ActivityIndicator(isAnimating: true)
+            } else {
+                VStack(alignment: .leading, spacing: 4) {
+                    Text(viewModel.dogBreedDetail?.name ?? "error")
```

Comment:
Instead of showing error `dogBreedDetail` is nil, better to show `no name`, `no description`. Having 0 for integers is fine though.